tests/jquery.flot-relativeTimeAxis.Test.js

```diff
@@ -54,50 +54,63 @@ describe('A Flot chart with relative time axes', function () {
     });
 
     it('shows time bigger than 1 second correctly', function () {
-        plot = createPlotWithRelativeTimeAxis(placeholder, [[[1, 1], [2, 2]]]);
+        plot = createPlotWithRelativeTimeAxis(placeholder, [[[0, 0], [1, 1], [2, 2]]]);
```

Comment:
the relative time doesn't always start from 0. It should start from the time of the first sample written. you could just write first a sample at 0 and then you could keep those tests.